src/Core/src/nuget/buildTransitive/Microsoft.Maui.Core.After.targets

```diff
@@ -7,6 +7,32 @@
 
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.Maui.Core.BundledVersions.targets" />
 
+  <!-- 
+    Check to make sure that .NET MAUI packages are not installed as .NET Standard
+    when they are installed into a .NET "Core" project. If this happens, it means
+    that the user has installed a .NET MAUI package that requires a later TFM than
+    what the project has.
+
+    Some examples:
+     - Installing a .NET 9 MAUI package into a .NET 8 project
+     - Installing a .NET 9 MAUI package into a very old .NET 9 installation
+
+    To skip this check, set $(SkipCheckForMauiMinimumSupportedWorkloadVersion) to true.
```

Comment:
This will need some documentation heads up @davidortinau  .

Reply:
Do we _want_ to advertise this option? It will only cause pain, and I just added this as a failsafe just in case.